source/apps/change-deployment-models.txt

```diff
@@ -0,0 +1,227 @@
+            to host your App's configuration data.
+
+            .. code-block:: json
+
+               {
+                 "deployment_model": "GLOBAL",
+                 "provider_region": "aws-us-east-1"
+               }
+
+            If you choose ``LOCAL``, choose a specific cloud provider
+            and region to deploy to. For a list of all available
+            options, see :ref:`Deployment Models & Regions
+            <deployment-model>`.
+
+            .. code-block:: json
+
+               {
+                 "deployment_model": "LOCAL",
+                 "provider_region": "azure-westus"
+               }
+
+         .. step:: Start the migration
+
+            To start a migration, call the :admin-api-endpoint:`Create a
+            Deployment Migration
+            <operation/adminCreateDeploymentMigration>` endpoint with
+            the deployment model and region you want to migrate to in
+            the request body.
+
+            .. code-block:: sh
+
+               curl -X PUT \
+                 https://realm.mongodb.com/api/admin/v3.0/groups/{groupId}/apps/{appId}/deployment_migration \
+                 -H 'Content-Type: application/json' \
+                 -H 'Authorization: Bearer <AccessToken>' \
+                 -d '{
+                   "deployment_model": "<DeploymentModel>",
+                   "provider_region": "<RegionID>"
+                 }'
+
+         .. step:: Monitor the migration status
+
+            To get the current status of a deployment migration, call
+            the :admin-api-endpoint:`Get a Deployment Migration
+            <operation/adminGetDeploymentMigration>`
+            endpoint.
+
+            .. code-block:: sh
+               
+               curl -X GET \
+                 https://realm.mongodb.com/api/admin/v3.0/groups/<groupId>/apps/<appId>/deployment_migration \
+                 -H 'Authorization: Bearer <AccessToken>'
+
+.. _deployment-migration-process:
+
+Deployment Migration Process
+----------------------------
+
+Deployment migrations move your App's data and configuration to one or
+more new regions in a series of stages. At each stage, the process
+migrates a portion of your App to the new model and cleans up any
+artifacts from the previous model.
+
+A deployment migration moves through the following stages in order:
+
+1. ``"started"``: the migration has been started
+
+#. ``"downtime"``: the App is unavailable while the migration is in progress
+
+#. ``"enabling_event_subscriptions"``: the App's Triggers and Device
+   Sync translators are being enabled
+
+#. ``"cleanup"``: deployment artifacts are being cleaned up
+
+#. The migration is complete and can be in one of two states:
+
+   - ``"successful"``: the migration completed successfully
+
+   - ``"failed"``: the migration failed
+
+The migration process migrates the following components of your App:
+
+- User accounts
+- Configuration files
+- Triggers
+- API Services
```

Comment:
> API Services

i see you did title case here..is 'API Services' now an Official Name that We Call These Things?

Reply:
Yeah that's the plan - it's not in the product anywhere but separate API services was confusing for folks